### DISCUSSION TITLE
One year celebration!

### DISCUSSION THREAD

**Discussion by dpriskorn** [LeMyst/WikibaseIntegrator#197] [General]
Hi, I created https://www.wikidata.org/wiki/Q108259643 and noticed that you forked around 4/8-20
Any ideas how to celebrate? :partying_face:

**Comment by dpriskorn** [comment]
I added it to https://archive.softwareheritage.org/swh:1:dir:2028088df742a2501e943a930be934048b6b4aff;origin=https://github.com/LeMyst/WikibaseIntegrator;visit=swh:1:snp:8613e27f620bd76fdb87c83a8fb28b435bd67782;anchor=swh:1:rev:9de8daff61111ee467fade6daeca729493bd20ad which seemed like a good start :)

**Reply by LeMyst** [reply]
Thank you, I'm not sure if WBI deserve to be archived in the current state, but you don't know what will happen in future and maybe this will be useful for someone in 10+ years. 😄

**Comment by LeMyst** [comment]
Hello @dpriskorn 
Thank you for the reminder, I totally forgot I forked WikidataIntegrator one year ago.
At first, it was simply to have more freedom, be oriented to Wikibase instead of Wikidata only and to have less wikidata-specific code inside the library.
After one year, I'm happy with the state for the project, the v0.11 seem to be viable and the huge rewrite in v0.12 is very promising.

To be honest, with all the job in v0.12, I think the celebration should occur at the release of v0.12.0 😉 

🎂 🎆
